plugins/item_tasks/plugin_tests/widgetsSpec.js

```diff
@@ -1,777 +1,777 @@
 girderTest.importPlugin('jobs', 'worker', 'item_tasks');
 // This is a unit test, so do not start the app
 
-
+var itemTasks;
 girderTest.promise.done(function () {
-    var itemTasks = girder.plugins.item_tasks;
+    itemTasks = girder.plugins.item_tasks;
```

Comment:
FYI, this is the only place in this commit that's not simply stripping a `$(function` and de-indenting. We need to preserve the previous behavior of waiting until the plugin is fully loaded (which is definitely true once `girderTest.promise` resolves) before being able to look it up.

The actual specs `describe` / `it` can of course still be defined right away, since they're not executed until the whole page is ready (which the the point of this whole commit).